Doc/library/shutil.rst

```diff
@@ -542,10 +542,11 @@ provided.  They rely on the :mod:`zipfile` and :mod:`tarfile` modules.
    Create an archive file (such as zip or tar) and return its name.
 
    *base_name* is the name of the file to create, including the path, minus
-   any format-specific extension. *format* is the archive format: one of
-   "zip" (if the :mod:`zlib` module is available), "tar", "gztar" (if the
-   :mod:`zlib` module is available), "bztar" (if the :mod:`bz2` module is
-   available), or "xztar" (if the :mod:`lzma` module is available).
+   any format-specific extension.
+   
+   *format* is the archive format: one of "zip" (if the :mod:`zlib` module
```

Comment:
To aid reviews, I would keep this paragraph in the same format as it was before (i.e., wrapped the same way except the first line). It'll be easier for reviews to more easily see the diffs that way.